packages/turf-bbox-clip/index.ts

```diff
@@ -37,7 +37,7 @@ import { lineclip, polygonclip } from "./lib/lineclip";
  */
 export default function bboxClip<
   G extends Polygon | MultiPolygon | LineString | MultiLineString,
-  P = GeoJsonProperties
+  P extends GeoJsonProperties = GeoJsonProperties,
```

Comment:
Is this duplicative? Does `P extends GeoJsonProperties` already imply default to `GeoJsonProperties`?

Reply:
TypeScript complains without it ☹:
<pre>
export default function bboxClip<
  G extends Polygon | MultiPolygon | LineString | MultiLineString,
  P = GeoJsonProperties
>(feature: Feature<G, P> | G, bbox: BBox) {
  const geom = getGeom(<ins>feature</ins>);
  ...
}
</pre>
> Argument of type 'G | Feature<G, P>' is not assignable to parameter of type 'G | Feature<G, GeoJsonProperties>'.
>   Type 'Feature<G, P>' is not assignable to type 'G | Feature<G, GeoJsonProperties>'.
>     Type 'Feature<G, P>' is not assignable to type 'Feature<G, GeoJsonProperties>'.
>       Type 'P' is not assignable to type 'GeoJsonProperties'.
>         Type 'P' is not assignable to type '{ [name: string]: any; }'.ts(2345)
> index.ts(40, 3): This type parameter might need an `extends { [name: string]: any; }` constraint.
> index.ts(40, 3): This type parameter might need an `extends GeoJsonProperties` constraint.